prov/verbs/src/verbs_info.c

```diff
@@ -803,6 +803,7 @@ static int fi_ibv_getifaddrs(struct dlist_entry *verbs_devs)
 			VERBS_WARN(FI_LOG_FABRIC,
 				   "inet_ntop failed: %s(%d)\n",
 				   strerror(errno), errno);
+			ret = -FI_EINVAL;
```

Comment:
`ret = -errno` ?

Reply:
inet_ntop can return EAFNOSUPPORT, but there's no FI_EAFNOSUPPORT. FI_EINVAL seemed as appropriate.

Reply:
I think we're okay using ret = -errno, since that provides a better indicator of the error.  We can expand the fi_errno defines as needed.